<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>683</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>228</red>
        <green>234</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>228</red>
        <green>234</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>120</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>192</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Главный экран</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/home-free-icon-font.png</normaloff>:/home-free-icon-font.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Задачи</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/list-check-free-icon-font.png</normaloff>:/list-check-free-icon-font.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_3">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>События</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/calendar-free-icon-font.png</normaloff>:/calendar-free-icon-font.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_4">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Проекты</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/apps-free-icon-font.png</normaloff>:/apps-free-icon-font.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_5">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Информация</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/bookmark-free-icon-font.png</normaloff>:/bookmark-free-icon-font.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>191</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="w_Calendar" native="true">
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,2" rowminimumheight="0,1,20">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 25pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>События</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>299</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="addEvent">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/plus-free-icon-font (1).png</normaloff>:/plus-free-icon-font (1).png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QCalendarWidget" name="calendarWidget"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="w_Task" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 25pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>Задачи</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>338</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Добавить задачу</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/plus-free-icon-font (1).png</normaloff>:/plus-free-icon-font (1).png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QListView" name="TaskView">
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="flow">
          <enum>QListView::TopToBottom</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="w_Project" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 75 25pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>Проекты</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>338</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="addProject">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Добавить задачу</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/plus-free-icon-font (1).png</normaloff>:/plus-free-icon-font (1).png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTreeView" name="ProjectView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="w_Info" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 75 25pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>Информация</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>338</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="addInfo">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Добавить задачу</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/plus-free-icon-font (1).png</normaloff>:/plus-free-icon-font (1).png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QListView" name="InfoView">
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="flow">
          <enum>QListView::TopToBottom</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
